samples/esb/esb_ptx/README.rst

```diff
@@ -30,20 +51,6 @@ Receiver
 The Receiver example listens for packets and sends an ACK when a packet is received.
 If packets are successfully received from the transmitter, the LED pattern will change every time a packet is received.
 
```

Comment:
You're describing two samples in this doc. I suggest splitting this and moving the respective docs to the respective sample readme.